content/secrets/04-walkthrough-step-01/copilot.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+In this section, it is important to match the names as described in order for the tutorial to work (so that it matches the format in the repository)
+
+Application Name: `ecsworkshop`
+
+Workload Type: `Load Balanced Web Service`
+
+Service Name: `todo-app`
+
+Deploy local test environment: `yes`
+
+
+```bash
+copilot init
+```
+
+{{< figure src="/images/secrets-copilot-init.gif" alt="Secrets Diagram" width="800px" >}}
+
+This will import values in the project located in `copilot/todo-app/manifest.yml`
+
```

Comment:
We should mention that it may take a few minutes and that the component that takes the most time is the aurora db deployment.

After they run copilot init, say something like "Since this is going to take a few minutes, let's spend that time understanding what is happening behind the scenes..."

Reply:
As i'm typing this review it took almost 13 minutes to deploy, which doesn't surprise me given that we're deploying an RDS environment. Definitely should give them some reading to do while it deploys :-)